ansible/configs/selinux-policy/post_software.yml

```diff
@@ -14,7 +14,7 @@
         - "Here are your credentials to access the lab environment:"
         - ""
         - "Password: {{ student_password | d(hostvars[groups.bastions.0].student_password)}}"
-        - "IP Address: {{ hostvars[groups.bastions.0].public_ip_address }}"
+        - "SSH Connection: {{ student_user }}@{{ hostvars[groups.bastions.0].public_ip_address }}"
```

Comment:
I guess this makes sense. But I would remove then the text from line 23 since it doesn't match your workshop documentation.